src/ontology/uberon-edit.obo

```diff
@@ -221742,39 +221739,35 @@ consider: UBERON:0009551
 
 [Term]
 id: UBERON:7500042
-name: line from the distal-most end of the longest phalax of the foot to the posterior-most part of the foot
-def: "An anatomical line of a hindfoot that connects the end of the longest digit to the posterior-most part of a foot." [https://orcid.org/0000-0003-2699-3066]
-intersection_of: UBERON:0006800 ! anatomical line
-intersection_of: connects UBERON:0009551 ! distal segment of digit
-intersection_of: connects UBERON:7500039 ! posterior of pes
+name: obsolete line from the distal-most end of the longest phalax of the foot to the posterior-most part of the foot
+def: "OBSOLETE An anatomical line of a hindfoot that connects the end of the longest digit to the posterior-most part of a foot." [https://orcid.org/0000-0003-2699-3066]
+comment: No longer part of the pattern design for lengths of lines. Can be found in FOVT as pes length without nail (FOVT:0000004).
```

Comment:
Uberon uses the 'consider' annotation property for terms to consider on obsoleted terms. I think you should add those using that annotation property, rather than mentioning them in the comment. It's okay if the consider points to another ontology term outside of Uberon.

See for example: 'obsolete atrium'

FYI - if you want to consider more than one term, you must have two consider annotations, like in the example above.